R/experiments.R

```diff
@@ -8,7 +8,7 @@
 #' @param name The name of the experiment to create.
 #' @param artifact_location Location where all artifacts for this experiment are stored. If
 #'   not provided, the remote server will select an appropriate default.
-#' @param client An MLFlow client. Defaults to `NULL` and will be auto-generated.
+#' @param client An MLFlow client. If not provided, the client is sourced from the `MLFLOW_TRACKING_URI` environment variable.
```

Comment:
updated what the default actually is. also, using `inheritParams` everywhere where it can be used to save from repeating this doc